app/src/org/commcare/android/adapters/EntityDetailAdapter.java

```diff
@@ -106,6 +116,10 @@ public View getView(int position, View convertView, ViewGroup parent) {
             dv.setParams(session, detail, entity, valid.get(position), detailIndex);
             dv.setCallListener(listener);
         }
+        if(modifier != null){
+            modifier.modifyEntityDetailView(dv);
+        }
+        dv.setLineColor(position % 2 != 0);
```

Comment:
It is always nice to wrap multi-operator expressions in parenthesis so that you don't need to think about operator precedence, (not that it is too hard in this case, just takes an extra few seconds for me to register what is going on).

For instance, ((position % 2) != 0)